pkg/stores/sqlpartition/queryparser/labels.go

```diff
@@ -0,0 +1,143 @@
+package queryparser
```

Comment:
Should this not retain its original license header?

Reply:
good point. I made a few changes but let me look into it